src/style.css

```diff
@@ -0,0 +1,123 @@
+html {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.info_title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.info_description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 0 0 16px;
+}
+
+.review_block {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.review_stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.review_star {
+  margin-right: 4px;
+  display: block;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+}
+
+.review_star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
use styles from stars project for all counts of stars

Reply:
the styles are the same as in the stars project